deltachat-ffi/deltachat.h

```diff
@@ -3558,6 +3616,41 @@ char*           dc_msg_get_filename           (const dc_msg_t* msg);
 char*           dc_msg_get_filemime           (const dc_msg_t* msg);
 
 
+/**
+ * Return file from inside an webxdc message.
+ *
+ * @memberof dc_msg_t
+ * @param msg The webxdc instance.
+ * @param filename The name inside the archive,
+ *     must be given as a relative path (no leading `/`).
```

Comment:
can we just ignore leading `/`?
turns out it is already ignored, so why write this here?

Reply:
forgot to update doc after fixing the `/` issue ;) will add an update now.